configure.ac

```diff
@@ -2567,7 +2567,6 @@ SURICATA_BUILD_CONF="Suricata Configuration:
   Landlock support:                        ${enable_landlock}
   Systemd support:                         ${enable_systemd}
 
-  Rust support:                            ${enable_rust}
```

Comment:
The definition of `enable_rust="yes" ` should also be removed ;-)

Reply:
Thankyou @catenacyber , didnt think of that